tests/system/datasources_test.go

```diff
@@ -0,0 +1,57 @@
+package system
+
+import (
+	"testing"
+)
+
+func TestDataSources(t *testing.T) {
+	t.Parallel()
+
+	s := NewSystemTest(t, "cases/datasources", nil)
+	s.Terraform.Apply()
+	defer s.Terraform.Destroy()
+
+	// Compare outputs of data and resource values (resource
+	// values have the '_expected' suffix)
+	checkOutput := func(key string) {
+		log.Debugf("Checking data source vs resource output for key: %s", key)
+		datasourceOutput := s.Terraform.Output(key)
+		resourceOutput := s.Terraform.Output(key + "_expected")
+
+		if datasourceOutput != resourceOutput {
```

Comment:
We tend to follow  a common pattern during tests when we want to test if two things are equal:

```
if v, want := 1, 2; v != want { 
    t.Errorf("thing was %v, expected %v", v, want) 
}
```

so most of `checkOutput` could be condensed to:
```
if v, want := s.Terraform.Output(key), s.Terraform.Output(key+"_expected"); v != want {
    t.Errorf("%s: value was '%s', expected '%s'", key, v, want)
}
```